test/DetectNetworkCards_spec.rb

```diff
@@ -3,37 +3,34 @@
 Yast.import "FcoeClient"
 
 describe Yast::FcoeClientClass do
-  before :each do
-    @fcoe = Yast::FcoeClientClass.new
-    @fcoe.main()
-
-    @test_class = @fcoe
+  subject { Yast::FcoeClient }
```

Comment:
well, previously it always reinit subject by calling main and it is gone now. Cannot be there some race conditions in different test order?

Reply:
Good point! Actually I do not know, but I tried
```shell
for n in {1..100}; do rspec --order rand test/*.rb ; done
```

and it was fine 100x :grinning: